patching/plugin-commands-patching/test/patch.test.ts

```diff
@@ -142,6 +142,116 @@ describe('patch and commit', () => {
     expect(fs.existsSync('node_modules/is-positive/license')).toBe(false)
   })
 
+  test('patch-commit multiple times', async () => {
+    const output = await patch.handler(defaultPatchOption, ['is-positive@1.0.0'])
+    const patchDir = getPatchDirFromPatchOutput(output)
+
+    // store patch files in a directory inside modules dir when not given editDir option
+    expect(patchDir).toContain(path.join('node_modules', '.pnpm_patches', 'is-positive@1.0.0'))
+    expect(path.basename(patchDir)).toBe('is-positive@1.0.0')
+    expect(fs.existsSync(patchDir)).toBe(true)
+
+    // sanity check to ensure that the license file contains the expected string
+    expect(fs.readFileSync(path.join(patchDir, 'license'), 'utf8')).toContain('The MIT License (MIT)')
+
+    {
+      console.debug('FIRST EDIT')
```

Comment:
do we need this logs? If we want to group some assertions, I guess we could used nested describe functions. The before scripts may be async

Reply:
These assertions must be in the same test. After all, this test tests the ability to run `patch-commit` multiple times.